utils/src/main/kotlin/LicenseFilenamePatterns.kt

```diff
@@ -90,23 +88,17 @@ object LicenseFilenamePatterns {
     }
 
     /**
-     * Return glob patterns which match files in [directory], in any ancestor directory of [directory] recursively
-     * and in any sub-directory of directory, if the filename matches any of the [filenamePatterns].
+     * Return glob patterns which match files in any parent directory of [directory] recursively, if the filename
+     * matches any of the [filenamePatterns].
      */
-    internal fun getFileGlobsForDirectoryAndAncestors(
+    internal fun getFileGlobsForParentDirectories(
```

Comment:
The terminology `ancestor` was used as opposed to `parent` in this file, IIUC consistently.
Now it's not consistent anymore. How about keeping ancestor?

Reply:
Done. Keeping Ancestor.